### PR TITLE
`prep_fringe`: load water mask into geometry file via `--water-mask`

### PR DESCRIPTION
**Pull request by bbuzz31**
simple change to allow watermask to be stored in geometry via prep_fringe.py.
I'm not sure how the CLI is working, but this did as expected for me.

### CODE REVIEW

File: src/mintpy/prep_fringe.py [hunk 1]
```diff
@@ -26,6 +26,76 @@
...
+
+
+def cmd_line_parse(iargs=None):
+    parser = create_parser()
+    inps = parser.parse_args(args=iargs)
+
+    # in case meta_file is input as wildcard
+    inps.metaFile = sorted(glob.glob(inps.metaFile))[0]
+
+    return inps
```
Comment (yunjunz) [inline src/mintpy/prep_fringe.py:98]: This part of change seems unnecessary given the current CLI setup.

**Comment by yunjunz** [comment]
The PR looks good to me now. @bbuzz31 could you try it again to confirm if it works as you expected?

**Comment by bbuzz31** [comment]
@yunjunz works for me, thanks!

**Review by yunjunz — APPROVED** [review]

**Comment by yunjunz** [comment]
> @yunjunz works for me, thanks!

Great, thanks for the PR!

**Pull request merged**